packages/flet/lib/src/controls/cupertino_app_bar.dart

```diff
@@ -62,7 +62,7 @@ class CupertinoAppBarControl extends StatelessWidget
           : null,
       trailing: trailingCtrls.isNotEmpty
           ? createControl(control, trailingCtrls.first.id, control.isDisabled,
-              parentAdaptive: parentAdaptive)
+              parentAdaptive: parentAdaptive) // todo: wrap in row?
```

Comment:
Why?

Reply:
In the conversion from `AppBar` to `CupertinoAppBar` when using `adaptive=True`, only the first trailing item is used/created - because the `CupertinoAppBar` allows only one child.
Was thinking of wrapping `AppBar.trailing` into a `Row`, then instead pass this row as child, making all the trailing controls visible.

Reply:
OK, let's keep it this way.